src/test/java/com/connexta/store/CreateProductTests.java

```diff
@@ -39,7 +39,7 @@
 @RunWith(SpringRunner.class)
 @SpringBootTest
 @AutoConfigureMockMvc
-public class CreateProductITests {
+public class CreateProductTests {
```

Comment:
❓ Is this the new naming scheme for ITests because they will eventually be in their own Itest directory?

Reply:
No, that was not the reason. I needed a way to only run the tests that use `GenericContainer` on Jenkins and not on CI. An easy way to do this was to rename the tests so that it's easy to skip the *ITest classes. Feel free to update this if you have a better idea.